modules/metering-store-data-in-s3.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * metering/configuring-metering/metering-configure-persistent-storage.adoc
+
+[id="metering-store-data-in-s3_{context}"]
+= Storing data in Amazon S3
+
+To use Amazon S3 for storage, edit the `spec.storage` section in the example `s3-storage.yaml` file below.
+
+[source,yaml]
+----
+apiVersion: metering.openshift.io/v1
+kind: MeteringConfig
+metadata:
+  name: "operator-metering"
+spec:
+  storage:
+    type: "hive"
+    hive:
+      type: "s3"
+      s3:
+        bucket: "bucketname/path/" <1>
+        region: "us-west-1" <2>
+        secretName: "my-aws-secret" <3>
+        # Set to false if you want to provide an existing bucket, instead of
+        # having metering create the bucket on your behalf.
+        createBucket: true <4>
+----
+<1> Specify the name of the bucket where you would like to store your data. You may optionally specify the path within the bucket.
+<2> Specify the region of your bucket.
+<3> The name of a secret in the metering namespace containing the AWS credentials in the `data.aws-access-key-id` and `data.aws-secret-access-key` fields. See the examples that follow for more details.
```

Comment:
@lamek, we should put a note about the key and Id filed need to be base64 encoded like (https://github.com/operator-framework/operator-metering/blob/master/Documentation/configuring-storage.md#storing-data-in-amazon-s3)

Reply:
Got it.